app/services/booking_responder/accept.rb

```diff
@@ -29,7 +29,9 @@ def book_to_nomis
     end
 
     def nomis_visit_creator
-      @nomis_visit_creator ||= CreateNomisVisit.new(visit)
+      @nomis_visit_creator ||= CreateNomisVisit.new(
+        visit, creator: staff_response.creator
```

Comment:
Is it also worth passing the visit as a hash for consistency / increased safety?

Reply:
No, using keywords argument everywhere is just insane clutter. Here `visit` is the main subject of the class which also bares 'Visit` in the class name, I'd rather remove the `creator` keyword argument